collector-sqs/src/main/java/zipkin/collector/sqs/SqsCollector.java

```diff
@@ -34,18 +34,19 @@
 import zipkin.internal.LazyCloseable;
 import zipkin.internal.Util;
 import zipkin.storage.StorageComponent;
-import static zipkin.internal.Util.*;
 
+import static zipkin.internal.Util.checkArgument;
 
-public final class AwsSqsCollector implements CollectorComponent, Closeable {
+
+public final class SqsCollector implements CollectorComponent, Closeable {
```

Comment:
Supernit: probably want to be consistent with SQS or Sqs. Latter is fine by me (I think sender would need to change)